test/actions/Create.js

```diff
@@ -50,19 +50,22 @@ describe('Actions::Create', () => {
         type: 'TEST_CREATE_PRE',
         instance: 'users',
         record: Map({ other: 'data' }),
-        uidField: configBase.uidField
+        uidField: configBase.uidField,
+        isAsync: false
       })).to.be.true
       expect(dispatchSpy.secondCall.calledWith({
         type: 'TEST_CREATE',
         instance: 'users',
         record: Map({ other: 'data' }),
-        uidField: configBase.uidField
+        uidField: configBase.uidField,
+        isAsync: false
       })).to.be.true
       expect(dispatchSpy.thirdCall.calledWith({
         type: 'TEST_CREATE_POST',
         instance: 'users',
         record: Map({ other: 'data' }),
-        uidField: configBase.uidField
+        uidField: configBase.uidField,
+        isAsync: false
       })).to.be.true
     })
```

Comment:
Well doesn't this test cover that? The action is created using `configBase` which doesn't have creator/updater/deleter configured (in each respective test file) and we check that `isAsync` is false.